src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,26 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
```

Comment:
Ok .. but I am skeptical that the issue @pgoodrich found occurs with the old line .. since we have been doing the "ETC" job sync forever without any issues (and the case he mentioned was under /etc/segmentation